javascript-practice/src/js/views/dashboardView.js

```diff
@@ -78,18 +115,115 @@ export default class DashboardView {
       const valueAddPassword = this.addPasswordEl.value;
       const valueAddConfirmPassword = this.addConfirmPasswordEl.value;
 
-      submitAddUser(
-        valueAddUserId,
-        valueAddFirstNameId,
-        valueAddLastNameId,
-        valueAddEmailId,
-        valueAddMobileNoId,
-        valueAddRole,
-        valueAddUserName,
-        valueAddPassword,
-        valueAddConfirmPassword
-      );
+      Object.values(this.addUserErrorEls).forEach((el) => {
+        el.textContent = '';
+      });
+
+      if (
```

Comment:
Your code is so complicated so please add comments to explain your code

Reply:
i done write comment